report_async/static/src/js/qweb_action_manager.js

```diff
@@ -0,0 +1,82 @@
+odoo.define('report_async.print', function(require) {
+    'use strict';
+
+    var ActionManager = require('web.ActionManager');
+    var core = require('web.core');
+    var framework = require('web.framework');
+    var Model = require('web.Model');
+    var Dialog = require('web.Dialog');
+
+    var _t = core._t;
+    var QWeb = core.qweb;
+
+    ActionManager.include({
+        ir_actions_report_xml: function(action, options) {
+            var self = this;
+            var _super = this._super;
+            var _action = _.clone(action);
+            var $content = $(QWeb.render("ReportAsyncConfiguration", {}));
+
+            // Popup for async Configuration
+            new Dialog(self, {
+                title: _t("Async Report Configuration ") + '(' + action['display_name'] + ')',
+                size : "medium",
+                buttons: [{
+                    text: _t("Print"),
+                    classes: 'btn-primary',
+                    close: true,
+                    click: function () {
+                        var is_report_async = this.$('#async_report_checker')
```

Comment:
Before we had a call to the backend that checked whether this report has the `is async` boolean set - could you still check for that, and let it determine whether to show this dialog or not? It's a bit much to have this dialog for any report in the system - I guess they will only enable it for the slow ones, otherwise it's too much clicking.